docs-src/src/components/BodyConfigCombat.jsx

```diff
@@ -662,6 +684,51 @@ class BodyConfigCombat extends PureComponent {
             </FormControl>
           </Grid>
 
+          <Grid item xs={4} sm={2} className={classes.formGrid}>
+            <FormControl disabled={!combatEnabled} margin='normal' fullWidth>
+              <InputLabel htmlFor='combatForceRetreatNode1' shrink={true} className={classes.reactSelectLabel}>
+                <Localize field='bodyConfig.combatForceRetreatNode1' />
+              </InputLabel>
+              <Select
+                className={classes.reactSelect}
+                simpleValue={true}
+                name='combatForceRetreatNode1'
+                value={combatForceRetreatNode1}
+                options={NODES}
+                onChange={value => this.setState({ combatForceRetreatNode1: value })}
+                disabled={!combatEnabled}
+                fullWidth />
+            </FormControl>
+          </Grid>
+          <Grid item xs={4} sm={1} className={classes.formGridButton}>
```

Comment:
Again, the button and collection Multi select can all be replaced by making the previous Select field the Multi field (and the source of truth).